components/CommunitiesCarousel.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Slider from "react-slick";
+
+class CommunitiesCarousel extends React.Component {
```

Comment:
It is no big deal, but this component doesn't _need_ to extend React.Component. We might use a plain function component here?

Reply:
I agree, let's avoid classes as much as possible, keeping our app running faster, also this is enforcing us to use hooks, and since a lot of people are learning, its good to learn the most up to date way to write React